client/shipping/woocommerce-services-item.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { Button, ExternalLink } from '@wordpress/components';
+import { Pill } from '@woocommerce/components';
+import { PLUGINS_STORE_NAME } from '@woocommerce/data';
+import { getAdminLink } from '@woocommerce/wc-admin-settings';
+
+const WooCommerceServicesItem: React.FC< {
+	pluginsBeingSetup: Array< string >;
+	onSetupClick: ( slugs: string[] ) => PromiseLike< void >;
+} > = ( { onSetupClick, pluginsBeingSetup } ) => {
+	const { createSuccessNotice } = useDispatch( 'core/notices' );
+
+	const isSiteConnectedToJetpack = useSelect( ( select ) =>
+		select( PLUGINS_STORE_NAME ).isJetpackConnected()
+	);
+
+	const handleSetupClick = () => {
+		onSetupClick( [ 'woocommerce-services' ] ).then( () => {
+			const actions = [];
+			if ( ! isSiteConnectedToJetpack ) {
+				actions.push( {
+					url: getAdminLink( 'plugins.php' ),
```

Comment:
Could we directly use the Jetpack connection URL? `getJetpackConnectUrl( queryArgs )`?

Reply:
I started along that path - but then I realized that it wouldn't work if Jetpack is not installed. If Jetpack is not installed, we would need to install it here. But once https://github.com/Automattic/woocommerce-services/pull/2439 is merged, we wouldn't need the jetpack plugin anymore.

For that reason, in order to avoid too much branching logic to maintain, I am just redirecting to the plugin page, where we display a banner recommending to install or connect to Jetpack 😄